doc/cli_vs_yum.rst

```diff
@@ -285,3 +285,96 @@ DNF will not magically replace a request for installing package ``X`` to install
 
 See the the related `Fedora bug 1096506
 <https://bugzilla.redhat.com/show_bug.cgi?id=1096506>`_ and `guidelines for renaming and obsoleting packages in Fedora <http://fedoraproject.org/wiki/Upgrade_paths_%E2%80%94_renaming_or_splitting_packages>`_.
+
+###############################################
+ Changes in DNF plugins compared to Yum plugins
+###############################################
+
+==================================  ========================================  ===============================
+Original Yum tool                   DNF command/option                        Package
+----------------------------------  ----------------------------------------  -------------------------------
+``yum-langpacks``                                                             ``dnf-langpacks``
+``yum-plugin-copr``                 `dnf copr`_                               ``dnf-plugins-core``
+``yum-plugin-fastestmirror``        ``fastestmirror`` option in `dnf.conf`_   ``dnf``
+``yum-plugin-fs-snapshot``                                                    ``dnf-plugins-extras-snapper``
+``yum-plugin-local``                                                          ``dnf-plugins-extras-local``
+``yum-plugin-merge-conf``                                                     ``dnf-plugins-extras-rpmconf``
+``yum-plugin-priorities``           ``priority`` option in `dnf.conf`_        ``dnf``
+``yum-plugin-remove-with-leaves``   ``dnf autoremove``                        ``dnf``
+==================================  ========================================  ===============================
+
+Plugins that have not been ported yet:
+
+``yum-plugin-aliases``,
+``yum-plugin-auto-update-debug-info``,
+``yum-plugin-changelog``,
+``yum-plugin-filter-data``,
+``yum-plugin-keys``,
+``yum-plugin-list-data``,
+``yum-plugin-post-transaction-actions``,
+``yum-plugin-protectbase``,
+``yum-plugin-ps``,
+``yum-plugin-puppetverify``,
+``yum-plugin-refresh-updatesd``,
+``yum-plugin-rpm-warm-cache``,
+``yum-plugin-show-leaves``,
+``yum-plugin-tmprepo``,
+``yum-plugin-tsflags``,
+``yum-plugin-upgrade-helper``,
+``yum-plugin-verify``,
+``yum-plugin-versionlock``
+
+Feel free to file a RFE for missing functionality if you need it.
```

Comment:
Give them a hint `https://github.com/rpm-software-management/dnf/wiki/Bug-Reporting#new-feature-request` (make the link visible from man page)